driver-mapping/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,25 @@
+#
```

Comment:
Didn't realize we didn't have one of these already.   Can you change rootLogger to INFO to match the one in driver-core?